packages/styles/src/molecules/product-title.css

```diff
@@ -0,0 +1,23 @@
+[data-fs-product-title] {
+    width: fit-content;
+    background-color: #cecece;
+    padding: 8px;
+    border-radius: 5px;
+}
+[data-fs-product-title] [data-fs-product-title-header] {
+    display: flex;
+    column-gap: .75rem;
+    justify-content: space-between;
+}
+[data-fs-product-title] [data-fs-product-title-header]:first-child {
+    font-size: 20px;
+    font-weight: 400;
+    line-height: 1.12;
+}
+[data-fs-product-title] [data-fs-product-title-header] [data-store-badge] {
+    white-space: nowrap;
+}
+[data-fs-product-title] [data-fs-product-title-addendum] {
+    margin-top: 1rem;
+    color: #5d666f;
+}
```

Comment:
Github is complaining about the new line.
<img width="270" alt="Screen Shot 2022-07-29 at 13 26 59" src="https://user-images.githubusercontent.com/11325562/181803375-ca241c38-e666-484a-8fb4-6ba3eb62f45c.png">